.github/workflows/terraform-deployment.yml

```diff
@@ -9,6 +9,8 @@ jobs:
   plan:
     name: Terraform plan
     runs-on: ubuntu-latest
+    env:
+      function_zip_path: "fap-server-${GITHUB_SHA::8}.zip"
```

Comment:
Warum heißt die Variable `function_zip_path`? Hier ändert sich ja nur der Name und nicht der Pfad, oder?

Reply:
Wir könnten sie auch `zip_name` oder so nennen - den Namen hat die Variable halt auch in terraform. Deshalb habe ich die einheitlich benannt.

Reply:
Ich verstehe, dann approve ich mal 🎉